### PR TITLE
Add sw notify

### PR DESCRIPTION
**Pull request by jaumef**
### Description:

Afegir documentació per les notificacions a clients ATR mitjançant l'assistent de notificació de casos pendents de notificar.

### Tasks:

- [x] Afegir documentació de l'assistent de notificació ATR
- [x] Afegir documentació de la configuració de les notificacions ATR
- [x] Fix menú lateral ATR (jerarquia de títols)
- [x] Traduïr

### Links:

* [Base/GestionATR#Notificacions a casos ATR](http://builds.gisce.net/powerp-docs/powerp_add_sw_notify/ca/base/atr/#notificacions-a-clients-de-casos-atr)
* [Base/GestionATR#Notificaciones a casos ATR](http://builds.gisce.net/powerp-docs/powerp_add_sw_notify/es/base/atr/#notificaciones-a-clientes-de-casos-atr)

**Links to images changed**
- http://builds.gisce.net/powerp-docs/powerp_add_sw_notify/ca/comer/atr/
- http://builds.gisce.net/powerp-docs/powerp_add_sw_notify/es/comer/atr/

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_sw_notify/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_sw_notify/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_sw_notify/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_sw_notify/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_sw_notify/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/base/atr.md [hunk 1]
```diff
@@ -618,8 +623,83 @@ correu a la fitxa PowerEmail.
 
 
 ###### Formulari notificació activació canvi de comercialitzadora
+
 ![](_static/atr/PowerEmailActivacio.png)
 
+### Notificacions a clients de casos ATR
```
Comment (gdalmau) [inline docs/base/atr.md]: (es) `cliente` → `clientes`

File: docs/base/atr.md [hunk 2]
```diff
@@ -618,8 +623,83 @@ correu a la fitxa PowerEmail.
...
+
+Hem desenvolupat un sistema que permet notificar als clients dels casos atr.
```
Comment (gdalmau) [inline docs/base/atr.md]: `atr` → `ATR`

File: docs/base/atr.md [hunk 3]
```diff
@@ -618,8 +623,83 @@ correu a la fitxa PowerEmail.
...
+
+Aquest sistema es basa amb plantilles PowerEmail i un assistent per enviar
```
Comment (gdalmau) [inline docs/base/atr.md]: `amb` → `en`

File: docs/base/atr.md [hunk 4]
```diff
@@ -618,8 +623,83 @@ correu a la fitxa PowerEmail.
...
+les notificacions als casos corresponents.
+
+Per defecte s'utilitza la variable `sw_mail_user_notification_on_activation` per
+saber si els nous passos creats s'han de notificar.
+
+Els passos per notificar es poden notificar manualment o bé,
+amb [l'assistent per notificar casos ATR](#assistent-per-notificar-casos-atr)
```
Comment (gdalmau) [inline docs/base/atr.md]: (es) `#assistente-para-notificar-casos-atr` → `#asistente-para-notificar-casos-atr`

File: docs/base/atr.md [hunk 5]
```diff
@@ -618,8 +623,83 @@ correu a la fitxa PowerEmail.
...
+
+!!! Warning "Atenció"
+    Els casos "pendents de notificar" només es marcaràn com a notificats si
```
Comment (gdalmau) [inline docs/base/atr.md]: `marcaràn` → `marcaran`
(es) `solo` → `sólo`

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_sw_notify/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/base/atr.md [hunk 6]
```diff
@@ -618,8 +623,83 @@ correu a la fitxa PowerEmail.
...
+ATR. Podem fer-ho desde la vista de casos ATR "**Menú OpenERP > Gestió ATR >
+Casos ATR**", prement sobre accions i escollint "_Enviar correu als casos
+ATR pendents de notificar_".
+
+![](_static/atr/WizardNotificarAccio.png)
+
+Seguidament l'assistent ens informarà sobre els passos marcats com "_pendent de
+notificar_" dels casos sel·leccionats.
+
+Si ens sembla correcte (podem repassar els passos de cada cas sel·leccionat),
```
Comment (gdalmau) [inline docs/base/atr.md]: `sel·leccionat` → `seleccionat`

File: docs/base/atr.md [hunk 7]
```diff
@@ -618,8 +623,83 @@ correu a la fitxa PowerEmail.
...
+podem procedir amb l'enviament. Prement el botó "_enviar correus pendents_":
+
+![](_static/atr/WizardNotificarPrevi.png)
+
+Aquest procés pot tardar una estona segons la quantitat de casos sel·leccionats.
```
Comment (gdalmau) [inline docs/base/atr.md]: `sel·leccionats` → `seleccionats`
`segun` → `según`

File: docs/base/atr.md (hunk 5 above)
Comment (jaumef) [reply docs/base/atr.md]: El `sólo`, ja no cal segons RAE

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_sw_notify/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_sw_notify/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by gdalmau** [comment]
[This commit](https://github.com/gisce/powerp-docs/pull/258/commits/cdd6dcc80eea4a47a7898c120d7120f15df029cb) doesn't have sense if you are modifying base/atr.md, not comer/atr.md. 

Revert the commit or fix the images of comer/atr.md if you want to include the commit.

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_sw_notify/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_sw_notify/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_sw_notify/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/base/atr.md [hunk 8]
```diff
@@ -618,8 +623,88 @@ correu a la fitxa PowerEmail.
...
+
+Hem desenvolupat un sistema que permet notificar als clients dels casos ATR.
+
+Aquest sistema es basa en plantilles PowerEmail i un assistent per enviar
+les notificacions als casos corresponents.
+
+Per defecte s'utilitza la variable `sw_mail_user_notification_on_activation` per
+saber si els nous passos creats s'han de notificar.
+
+Els passos per notificar es poden notificar manualment o bé,
```
Comment (gdalmau) [inline docs/base/atr.md]: `manualment, o bé amb l'assistent [...]`

File: docs/base/atr.md [hunk 9]
```diff
@@ -618,8 +623,88 @@ correu a la fitxa PowerEmail.
...
+
+Es poden trobar filtrant amb el nom de plantilla "_ATR {nom_del_cas}{nom_del_pas}:_".
+Per exemple: `ATR A301:`.
+
+S'ha de tenir en compte que el `object` que s'utilitza és el cas ATR.
+
+#### Assistent per notificar casos ATR
+
+Aquest assistent ens permet notificar de forma agrupada diversos passos i casos
+ATR. Podem fer-ho desde la vista de casos ATR "**Menú OpenERP > Gestió ATR >
```
Comment (gdalmau) [inline docs/base/atr.md]: (ca) `desde` → `des de`

**Comment by gdalmau** [comment]
Images from comer/atr work :+1: 

Changes these 2 little things and we can merge.

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_sw_notify/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Pull request merged**